MamboBryan/Randomy
# #166156859 Create landing activity for users

#### What does this PR do?
It creates a landing page for the users to access the app 
#### Description of Task to be completed?
* Adds welcome not and app description
* Adds app logo at the center of the screen
* Adds next button to navigate to the other activity
* Changes app background and button background
#### How should this be manually tested?
* Clone the repo into your machine
* Open app with android studio and wait to complete the build
* Run the app into your desired device
* After complete run the LandingActivity will appear on your device screen
* Take note of the above tasks completed
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166156859
#### Screenshots (if appropriate)
![LandingActivity](https://user-images.githubusercontent.com/40160345/58200640-e00ad880-7cdb-11e9-86c2-3ab13db06d98.PNG)
